lib/httplib/httpheaders.go

```diff
@@ -21,51 +21,77 @@ package httplib
 import (
 	"fmt"
 	"net/http"
+	"regexp"
 	"sort"
 	"strings"
 	"time"
 
-	"golang.org/x/exp/maps"
-
 	"github.com/gravitational/teleport/api/client/proto"
 )
 
-var defaultContentSecurityPolicy = map[string]string{
-	"default-src": "'self'",
+type cspMap map[string][]string
+
+var defaultContentSecurityPolicy = cspMap{
+	"default-src": {"'self'"},
 	// specify CSP directives not covered by `default-src`
-	"base-uri":        "'self'",
-	"form-action":     "'self'",
-	"frame-ancestors": "'none'",
+	"base-uri":        {"'self'"},
+	"form-action":     {"'self'"},
+	"frame-ancestors": {"'none'"},
 	// additional default restrictions
-	"object-src": "'none'",
-	"img-src":    "'self' data: blob:",
-	"style-src":  "'self' 'unsafe-inline'",
+	"object-src": {"'none'"},
+	"img-src":    {"'self'", "data:", "blob:"},
+	"style-src":  {"'self'", "'unsafe-inline'"},
 }
 
-var defaultFontSrc = map[string]string{"font-src": "'self' data:"}
+var defaultFontSrc = cspMap{"font-src": {"'self'", "data:"}}
+var defaultConnectSrc = cspMap{"connect-src": {"'self'", "wss:"}}
 
-var stripeSecurityPolicy = map[string]string{
+var stripeSecurityPolicy = cspMap{
 	// auto-pay plans in Cloud use stripe.com to manage billing information
-	"script-src": "'self' https://js.stripe.com",
-	"frame-src":  "https://js.stripe.com",
+	"script-src": {"'self'", "https://js.stripe.com"},
+	"frame-src":  {"https://js.stripe.com"},
+}
+
+var wasmSecurityPolicy = cspMap{
+	"script-src": {"'wasm-unsafe-eval'"},
+}
+
+func deduplicate(elements []string) []string {
```

Comment:
We do have `utils.Deduplicate`, but if you're prefer to avoid the import, this is fine too.